### ISSUE TITLE
doesn't currently work

### ISSUE THREAD

**Issue by jameski83** [mockingbirdconsulting/HashicorpAtHome#4]
When I try to do vault operator unseal I get an error unreachable on 127.0.0.1, even if I try and change the config files in /etc/consul vault and nomad to my actual IP address, only consul commands work. Nomad and Vault commands do not work.

**Comment by proffalken** [comment]
@jameski83 thanks for logging this, I note that you've closed it already, is the problem solved?

I'm happy to work through this with you if needed.

Thanks in advance :)

**Comment by jameski83** [comment]
Hi, thanks for getting back to me. To be honest i've tried to get a nomad/consul/vault cluster to work for a while - I also tried https://github.com/perrymanuk/hashi-homelab without success (can't say there is much useful documentation about that is directed at new users to the hashicorp stack). 

I closed the issue because it is probably something I'm doing rather than an issue with your guides. If you do have any time to assist then I would be grateful. 

I have a fresh debian VM that I am trying deployment on again. I have prepared everything up to step 6, run the playbook so I can generate the vault token and when I try and run vault operator init on the VM, I get the error "Error initializing: Put https://127.0.0.1:8200/v1/sys/init: dial tcp 127.0.0.1:8200: connect: connection refused". 

On the playbook run I also get this error ;

TASK [brianshumate.vault : Check Vault installation] **************************************
fatal: [homeserver01.consul]: FAILED! => {
    "changed": false,
    "cmd": "command -v vault",
    "rc": 2
}

MSG:

[Errno 2] No such file or directory

**Comment by jameski83** [comment]
I have managed to get a bit further, although I can't use any of the cli commands, even if I use the -address command within the VM I get the error above. I had to create the token via the vault web ui. I can get to the nomad & vault web ui but the consul web ui does not load. I ran the homeassistant job and it is active and the url homeassistant.service.homelab pings from the vm but when I change my dns to the VM I can't ping it and external DNS doesn't seem to be working either when set to the VM. 

So i'm not sure if consul is working properly - the consul service is 'active' though it has this error in the logs :

"Mar 03 20:21:16 nomad consul[963]:     2020/03/03 20:21:16 [ERR] memberlist: failed to receive: Encryption is configured but remote state is not encrypted"

Looks like a config issue perhaps?
